test/functional/page_objects/common_page.ts

```diff
@@ -344,6 +344,10 @@ export function CommonPageProvider({ getService, getPageObjects }: FtrProviderCo
       await browser.pressKeys(browser.keys.ENTER);
     }
 
+    async pressTabKey() {
+      await browser.pressKeys(browser.keys.TAB);
+    }
+
     // Pause the browser at a certain place for debugging
     // Not meant for usage in CI, only for dev-usage
     async pause() {
```

Comment:
this is now completely off topic, but I was just excited to find a pause function for the functional tests 🥳